log.go

```diff
@@ -61,6 +64,19 @@ func NewSrvLog() *SrvLog {
 	return obj
 }
 
+// NewNopSrvLog returns a new instance of a SrvLog object that doesn't log to disk.
+func NewNopSrvLog() *SrvLog {
+	obj := &SrvLog{
+		subs:   make(map[string]*LogService),
+		fields: make(map[string]interface{}),
+	}
+	logNop := log.NewLogBuffer(1)
+	obj.AddLog("debug", logNop)
+	obj.AddLog("error", logNop)
+	obj.AddLog("info", logNop)
+	return obj
+}
+
```

Comment:
Not sure I understand when/how this should get used. 

If the intent is to do the same thing as NewSrvLog but without the disk-based logging, why not use the existing logBuffer object (which feeds the in-memory log buffer you can retrieve via the REST debug apis)? If the intent is to be a total no-op then maybe just feed it log.NullLogger from the log package itself?

Reply:
Yeah, sorry. I'll strip this bit and put it in a different PR. The goal was to make a logger that would basically do nothing with any logs that came through. Using a NullLogger sounds like exactly what I want!

Reply:
Oh! Actually, the NullLogger doesn't implement log.LogNotify, so if we want to make that change we'll have to get the NullLogger to implement that interface first.